server/repositories/__tests__/hubMenuRepository.spec.js

```diff
@@ -1,37 +1,7 @@
 const { hubMenuRepository } = require('../hubMenu');
-const topicsResponse = require('../../../test/resources/tagsContent.json');
 const categoryMenuResponse = require('../../../test/resources/categoryMenuResponse.json');
-const vocabularyResponse = require('../../../test/resources/vocabularyResponse.json');
 
 describe('hubMenuRepository', () => {
-  describe('topicsMenu', () => {
-    it('returns a topicsMenu', async () => {
-      const httpClient = {
-        get: jest.fn().mockReturnValue(topicsResponse),
-      };
-
-      const repository = hubMenuRepository(httpClient);
-      const expected = [
-        { linkText: 'Bat', href: '/tags/1', id: '1', description: undefined },
-        { linkText: 'Baz', href: '/tags/0', id: '0', description: undefined },
-      ];
-      const result = await repository.tagsMenu();
```

Comment:
Deleted these 2 tests, they were testing internal methods.

* This test isn't testing what it claims to be testing.
* The other test is very basic happy path which will be rewritten in a follow up (with additional cases)